.spelling

```diff
@@ -731,6 +731,8 @@ wildcard
 wildcarded
 wildcards
 workgroup
+WorkloadEntry
+WorkloadGroup
```

Comment:
Don't add resource kinds as spelling exceptions. Wrap there uses in the doc in back-ticks (e.g., `WorkloadEntry`) instead.

```suggestion
```